modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PartitionMeta.java

```diff
@@ -17,27 +17,29 @@
 
 package org.apache.ignite.internal.pagememory.persistence;
 
+import static org.apache.ignite.internal.pagememory.PageIdAllocator.FLAG_AUX;
+import static org.apache.ignite.internal.pagememory.util.PageIdUtils.pageId;
+
 import java.lang.invoke.MethodHandles;
 import java.lang.invoke.VarHandle;
 import java.util.UUID;
-import java.util.concurrent.atomic.AtomicIntegerFieldUpdater;
+import org.apache.ignite.internal.pagememory.persistence.io.PartitionMetaIo;
 import org.apache.ignite.internal.tostring.S;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Partition meta information.
  */
 public class PartitionMeta {
-    private static final AtomicIntegerFieldUpdater<PartitionMeta> PAGE_COUNT_UPDATER = AtomicIntegerFieldUpdater.newUpdater(
-            PartitionMeta.class,
-            "pageCount"
-    );
+    private static final VarHandle PAGE_COUNT;
```

Comment:
I know that comments for private stuff are not mandatory, but you could at least add a link to a field. Just saying

Reply:
Are you serious?

Reply:
I said "could"

Reply:
Then I would leave it as is.